client/src/app/pages/search/components/SearchMenu.tsx

```diff
@@ -133,24 +109,44 @@ function useAllEntities(filterText: string) {
   }));
 
   const transformedVulnerabilities: IEntity[] = vulnerabilities.map((item) => ({
-    id: item.identifier,
+    id: `vulnerability-${item.identifier}`,
     title: item.identifier,
     description: item.description?.substring(0, 75),
     navLink: `/vulnerabilities/${item.identifier}`,
     type: "Vulnerability",
     typeColor: "orange",
   }));
 
-  tmpArray.push(
+  const filterTextLoweCase = filterText.toLowerCase();
```

Comment:
@carlosthe19916 Can you please change to `filterTextLowerCase`? Sorry but the typo will drive me crazy. ;D